Pipfile

```diff
@@ -29,7 +29,7 @@ psycopg2-binary = "*"
 cryptography = "*"
 peewee = "*"
 pyyaml = "*"
-python-magic = "*"
+python-magic-bin = "==0.4.14"
```

Comment:
I read up on python-magic and the related packages. Looks like `python-magic-bin` has DLLs for windows but not for other platforms. There is another package `python-magic-debian-bin` that has pre-compiled binaries for debian, mac and windows. 
Given that this may break on other operating systems, can you put a specifier to only install on windows? 
Reference: https://pipenv.pypa.io/en/latest/advanced/#specifying-basically-anything

So the change has to be:

    python-magic = {version = "*", sys_platform = "!= 'win32"}
    python-magic-bin = {version = "==0.4.14", sys_platform= "=='win32'"}

Once you make this change, I'll test on Linux and Mac.